dotnet/src/webdriver/CapabilityType.cs

```diff
@@ -161,6 +161,11 @@ public static class CapabilityType
         /// </summary>
         public static readonly string WebSocketUrl = "webSocketUrl";
 
+        /// <summary>
+        /// Capability name used to get a value indicating whether files may be downloaded from remote end.
+        /// </summary>
+        public static readonly string EnableDownloads = "se:downloadsEnabled";
```

Comment:
Should we ad it to `KnownSpecCompliantCapabilityNames`? I don't know, just to double check.

Reply:
Don't need to, the code also allows anything with a colon as valid by default.